posthog/models/notebook/notebook.py

```diff
@@ -21,6 +21,7 @@ class Notebook(FileSystemSyncMixin, RootTeamMixin, UUIDModel):
     content: JSONField = JSONField(default=None, null=True, blank=True)
     text_content = models.TextField(blank=True, null=True)
     deleted = models.BooleanField(default=False)
+    hidden = models.BooleanField(default=False)
```

Comment:
Wonder if it's worth making this an enum with types "default" and "internal"? Would allow us to add things like "public" in future. I know I originally pitched "hidden" but it's not very descriptive and I'd worry people would overload it with other features